packages/govern/internal/actions/QueuesForDAOAction.ts

```diff
@@ -0,0 +1,44 @@
+import { isAddress } from '@ethersproject/address'
+import { DocumentNode } from 'graphql'
+import AbstractAction from './lib/AbstractAction'
+import queuesForDAO from '../clients/graphql/queries/queuesForDAO'
+
+/**
+ * @class QueuesForDAOAction
+ */
+export default class QueuesForDAOAction extends AbstractAction {
+  /**
+   * Contains the GraphQL query of the current action
+   *
+   * @var {string} gqlQuery
+   *
+   * @protected
+   */
+  protected gqlQuery: DocumentNode = queuesForDAO
+
+  /**
+   * @param {Object} parameters
+   *
+   * @constructor
+   */
+  constructor(parameters: { address: string }) {
```

Comment:
Should we have a `type Address = string` like in the Govern Server package?

Reply:
Will do that 👌